### PR TITLE
DACT-1382: Added english localisation for etag stop screen

### PR DESCRIPTION
**Pull request by cdrainCH**
JIRA: https://companieshouse.atlassian.net/jira/software/c/projects/DACT/boards/492?selectedIssue=DACT-1382
English localisation for etag stop screen

Page with english localisation:

Screenshot 2024-02-16 at 12 09 56

### CODE REVIEW

File: locales/cy/stop-page.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+    "stopPageETAGPageHeader": "To be translated",
+    "stopPageETAGPageBody1": "To be translated",
+    "stopPageETAGPageBody2": "To be translated ",
+    "stopPageETAGStartServiceAgainLink": "To be translated",
+    "stopPageETAGContactUsLink": "To be translated",
```
Comment (jbestwickch) [inline locales/cy/stop-page.json:6]: I don't think ETAG needs to be capitalised here

File: locales/en/stop-page.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+    "stopPageETAGPageHeader": "Someone has already made updates for this director",
+    "stopPageETAGPageBody1": "Since you started using this service, someone else has submitted an update to this director's details.",
+    "stopPageETAGPageBody2": "If you still need to submit this update, you'll need to ",
```
Comment (jbestwickch) [inline locales/en/stop-page.json:4]: Same as above

File: locales/cy/stop-page.json (hunk 1 above)
Comment (cdrainCH) [reply locales/cy/stop-page.json]: Fixed

File: locales/en/stop-page.json (hunk 2 above)
Comment (cdrainCH) [reply locales/en/stop-page.json]: Fixed

**Review by jbestwickch — APPROVED** [review]

**Pull request merged**